mirgecom/steppers.py

```diff
@@ -27,8 +27,34 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 """
+
+import numpy as np
 from logpyle import set_dt
 from mirgecom.logging_quantities import set_sim_state
+from pytools import memoize_in
+from arraycontext import freeze, thaw
+
+
+def compile_timestepper(actx, timestepper, state, rhs):
+    """Create lazy evaluation version of the timestepper."""
+    @memoize_in(actx, ("mirgecom_compiled_operator",
+                       timestepper, rhs))
+    def get():
+        return actx.compile(lambda y, t, dt: timestepper(state=y, t=t,
+                                                         dt=dt,
+                                                         rhs=rhs))
+
+    return get()
```

Comment:
Definitely not a big deal, but this came in as a comment on one of my PRs.  
```suggestion
    def get_timestepper():
        return actx.compile(lambda y, t, dt: timestepper(state=y, t=t,
                                                         dt=dt,
                                                         rhs=rhs))

    return get_timestepper()
```